bears/js/JSHintBear.py

```diff
@@ -308,6 +306,12 @@ def generate_config(filename, file,
             This option is used to specify the ECMAScript version to which the
             code must adhere to.
         """
+        # Assume that when es_version is bool, it is intended for the
+        # deprecated use_es6_version
+        if es_version is True:
+            es_version = 6
+        elif es_version is False:
+            es_version = 5
```

Comment:
I haven't gone into the dept of reading it and understanding what it does
exactly, but it did give me an error. I've filed an issue here
https://github.com/coala-analyzer/coala/issues/2715

On Sat, Sep 3, 2016 at 9:36 PM, Lasse Schuirmann <notifications@github.com>
wrote:

> In bears/js/JSHintBear.py
> <https://github.com/coala-analyzer/coala-bears/pull/747#discussion_r77438715>
> :
>
> > @@ -308,6 +306,12 @@ def generate_config(filename, file,
> >              This option is used to specify the ECMAScript version to which the
> >              code must adhere to.
> >          """
> > +        # Assume that when es_version is bool, it is intended for the
> > +        # deprecated use_es6_version
> > +        if es_version is True:
> > +            es_version = 6
> > +        elif es_version is False:
> > +            es_version = 5
>
> isn't the conversion run on any given valu?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/coala-analyzer/coala-bears/pull/747/files/09efa41e64d6a88201c94ad2b0bff9a8f6ea52ed..5fd63b2490955fd31c1ea01d950139a6836fab10#r77438715>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ACGUp_Dm1Gw_ubLMcDJ5-PjpREWFzk4Jks5qmZrsgaJpZM4JxwAR>
> .
>